### PR TITLE
Streamline, proper documentation, cleanup

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by tboerger** [comment]
@bradrydzewski I have applied all comments from the streamline PRs.

File: main.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func main() {
 		cli.StringFlag{
 			Name:   "webhook",
 			Usage:  "slack webhook url",
-			EnvVar: "SLACK_WEBHOOK",
+			EnvVar: "SLACK_WEBHOOK,PLUGIN_WEBHOOK",
```
Comment (appleboy) [inline main.go]: 👍 What time do you merge this PR and update docker image?

Nice Job.

We need `environment` section to provide webhook url In `0.5` version now.

```
pipeline:
  slack:
    channel: drone
    username: drone
    environment:
      - SLACK_WEBHOOK=https://hooks.slack.com/services/xxxxxx

Comment (tboerger) [inline main.go]: It will be merged when brad is fine with it ;)

**Comment by estliberitas** [comment]
Any update on this?

**Comment by donny-dont** [comment]
LGTM

**Comment by Bugagazavr** [comment]
LGTM

**Pull request merged**